lib/checks/hasUnsupportedNodes.js

```diff
@@ -1,30 +1,29 @@
-const Node = require('../node');
+const {query} = require('math-nodes');
+
 const resolvesToConstant = require('./resolvesToConstant');
 
 function hasUnsupportedNodes(node) {
-  if (Node.Type.isParenthesis(node)) {
```

Comment:
I don't really know how `hasUnsupportedNodes` is used.  My assumption from the name of the function is that it's some sort of validator that checks for malformed ASTs.  @evykassirer could you describe what it's used for in the codebase?